CourseSketchProjects/CourseSketchProxyServer/src/main/java/coursesketch/serverfront/ProxyConnectionState.java

```diff
@@ -18,9 +18,16 @@ public ProxyConnectionState(final String inputKey) {
     }
 
     /**
-     * @return the user id of this connection.
+     * @return the authentication id of the user who logged in (its auth id)
      */
-    /* package-private */String getUserId() {
+    /* package-private */ String getAuthId() {
+        return getServerAuthId();
+    }
+
+    /**
+     * @return the identification id of the user who logged in (its user id)
+     */
+    /* package-private */ String getUserId() {
```

Comment:
yes.  it lets you know that only package access is allowed.  (To make sure it is not a mistake)